### ISSUE TITLE
Standardize the way validity dates are sent out from the services

### ISSUE THREAD

**Issue by camba1** [camba1/gotemp#4]
Currently, depending on wether the DB is SQL or No SQL, the validity dates and the modification fields are been sent out to the client either inline or as part of an object within the dataset. This is due to how the fields are mapped in the porto definition.
When creating the services we switched midway the way we handled those fields in the porto definitions to use imported global definitions but never went back to change it in the promotion and customer services.
Bottom line, all services should handle those fields the same way It is done in the product and customer services, independent of how they are stored in the DB
